datasets/sondehub-telemetry.yaml

```diff
@@ -0,0 +1,55 @@
+Name: SondeHub Radiosonde Telemetry
+Description: >
+  SondeHub Radiosonde telemetry contains global radiosonde (weather balloon) data captured by SondeHub from our
+  participating radiosonde_auto_rx receiving stations. Currently 313 receiver stations are providing
+  data for an average of 384 radiosondes a day.
+
+  Data contains telemetry frames in Universal Sonde Telemetry format, including gps position, type,
+  altitude, pressure, temperature and humidity.
+Documentation: https://github.com/projecthorus/sondehub-infra/wiki/Amazon-Open-Data
+Contact: Michaela Wheeler <vk3fur@sondehub.org>
+ManagedBy: SondeHub
+UpdateFrequency: Data is updated as we receive it
+Tags:
+  - climate
```

Comment:
please add `aws-pds` as a tag. We use this to indicate participation in the Open Data Sponsorship Program